docs/adr/rollup.md

```diff
@@ -8,7 +8,6 @@ This document outlines the architectural decisions made for the rollup mode of a
     - This trait is a template for the most important chain-related primitives used in Angstrom: `Block`, `BlockHeader`, `BlockBody`, `SignedTx`, `Receipt`
     - The default implementation is always [`EthPrimitives`](https://reth.rs/docs/reth/primitives/struct.EthPrimitives.html)
     - If needed, can be overridden to use [`OpPrimitives`](https://reth.rs/docs/op_reth/primitives/struct.OpPrimitives.html)
-    TODO: Add which crates are affected by this
```

Comment:
I don't think adding it here provides much value. IMO the PR https://github.com/SorellaLabs/angstrom/pull/569 will provide sufficient context.